### ISSUE TITLE
Menu option to clear queue in playlist screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#356]
```
What steps will reproduce the problem?
1. queue up songs
2. want to clear queue
3. not find an option to clear queue

What is the expected output? What do you see instead?
Right now, the menu doesn't do anything in the "music playlist" window - it 
would be nice to have a "clear playlist" menu option (and if you really want to 
get fancy, and this is probably a separate issue, save/load playlist). In other 
remote apps I've seen, "playing" a song (as compared to queuing it) sets the 
playlist to that one song. I like that xbmc remote doesn't do that, but would 
like some way of clearing the playlist accordingly.


What version of XBMC Remote are you using?
v0.8.3-beta1, rev 686


Which Android device are you using (and which ROM, if custom)?
Motorola Droid, 2.1


Which XBMC revision are you running and on which platform?
9.11 CVS



Please provide any additional information below.

```

Original issue reported on code.google.com by `davef....@gmail.com` on 12 Aug 2010 at 3:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `phree...@gmail.com` on 3 Sep 2010 at 1:02
- Changed state: **Accepted**
- Added labels: **Milestone-Release1.0**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Okay - in the interest of learning how to program android stuff better, and 
under the expectation that the likelihood of this happening goes up 
significantly if I help this along, I've gone ahead and created a patch that 
seems to work. I'm not particularly skilled in java in general, or android 
development in particular, so it's highly likely I've violated various 
important conventions, and would be happy to rework the patch as needed. I used 
the android stock "clear/cancel" icon, but wasn't sure if I'm referencing it 
the right way, or if I've properly separated out the logic to fit the 
presentation/business logic separation in the code. :) so, hopefully this isn't 
too bad.

Basically, the logic I implemented was a new menu item for the playlist screen, 
that when selected stops whatever's currently playing, tells xbmc to clear the 
playlist, then does the various gui cleanups to get the playlist screen back to 
"no items selected". It should properly handle both playing and not playing 
states, and items in the playlist and no items in the playlist. Anything else I 
can do, please let me know.
```

Original comment by `davef....@gmail.com` on 10 Oct 2010 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
heh. always helps to actually include the attachment.
```

Original comment by `davef....@gmail.com` on 10 Oct 2010 at 10:58

Attachments:
- [i356.patch](https://storage.googleapis.com/google-code-attachments/android-xbmcremote/issue-356/comment-3/i356.patch)
